components/appointments/id/index.tsx

```diff
@@ -22,7 +26,7 @@ import { getAppointmentWithAID } from '@/functions/server-actions/appointment';
 import { AppointmentType } from '@/models/Appointment';
 import { renderChip } from '@/components/ui/data-table/cell-renderers';
 import { CellRenderer } from '@/components/ui/cell-renderer';
-import { format } from 'date-fns';
+import { format, formatDate } from 'date-fns';
```

Comment:
instead of using `formatDate` use `format` only
```suggestion
import { format } from 'date-fns';
```